readme.md

```diff
@@ -48,14 +48,26 @@ Milliseconds before timing out.
 
 Passing `Infinity` will cause it to never time out.
 
-##### message
+##### error
 
-Type: `string | Error`\
+Type: `string | Error | false`\
 Default: `'Promise timed out after 50 milliseconds'`
 
-Specify a custom error message or error.
+Specify a custom error message or error to throw when it times out:
```

Comment:
This needs to be changed in the TS comment too.

Reply:
Now if only there was a way to automate this 😜 

Done